src/Ombi/ClientApp/app/requests/movierequests.component.html

```diff
@@ -78,40 +78,42 @@ <h4 class="request-title">{{request.title}} ({{request.releaseDate | date: 'yyyy
                     </a>
                 </div>
                 <br />
-                <div>
-                    <span>{{ 'Requests.RequestedBy' | translate }} </span>
-                    <span *ngIf="!isAdmin">{{request.requestedUser.userName}}</span>
-                    <span *ngIf="isAdmin && request.requestedUser.alias">{{request.requestedUser.alias}}</span>
-                    <span *ngIf="isAdmin && !request.requestedUser.alias">{{request.requestedUser.userName}}</span>
-                </div>
-                <div>
-                    <span>{{ 'Requests.Status' | translate }} </span>
-                    <span class="label label-success" id="requestedStatusLabel">{{request.status}}</span>
-                </div>
+                <div class="request-info">
```

Comment:
Created a wrapper to enclose the request info for movies and added a class to each item label